lib/bolt/cli.rb

```diff
@@ -551,6 +557,110 @@ def list_groups
       outputter.print_groups(groups)
     end
 
+    def new_plan(plan_name)
+      @logger.warn("Command 'bolt plan new' is experimental and subject to changes.")
```

Comment:
"No purchase necessary. Must be 18 or older to win." This reminds me of the 'fine-print' in a radio sweepstakes.